packages/store-ui/src/atoms/Bullets/Bullets.tsx

```diff
@@ -0,0 +1,44 @@
+import type { MouseEvent } from 'react'
+import React from 'react'
+
+import Button from '../Button'
+
+export interface BulletsProps {
+  totalQuantity: number
+  activeBullet: number
+  onClick: (e: MouseEvent, bulletIdx: number) => void
+  testId?: string
```

Comment:
can you write a comment with a description of each prop? it's useful to storybook props description and IDE IntelliSense.

Reply:
For sure! 😄